addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -226,9 +226,17 @@ QUnit.test('mailing channel composer: basic rendering', async function (assert)
 QUnit.test('add an emoji', async function (assert) {
     assert.expect(1);
 
+    this.data['mail.channel'].records.push({
+        channel_type: 'channel',
+        id: 20,
+        name: 'General',
```

Comment:
remove fields that are not necessary for the test, and use a generic thread if possible (`res.partner` for example) instead of channel which is a very special kind of thread

Reply:
I tried the res.partner: 
```javascript
    this.data['res.partner'].records.push({
        id: 20,
    });
    await this.start();
    const thread = this.env.models['mail.thread'].findFromIdentifyingData({
        id: 20,
        model: 'res.partner',
    });
```
But it's not working, thread is undefined for some reason

Reply:
Oh yes the thread will not be fetched at init this way, until the corresponding chatter is loaded. And we probably don't want to make this a chatter test, and having to fetch the thread manually feels annoying too.
So I guess it's fine to keep channel for now, even if counting on channels to be auto-loaded doesn't feel very reliable.

Reply:
@seb-odoo I guess we can merge this then ?